addons/hr/models/hr_employee.py

```diff
@@ -0,0 +1,151 @@
+    identification_id = fields.Char(string='Identification No')
+    gender = fields.Selection([
+         ('male', 'Male'),
+         ('female', 'Female'),
+         ('other', 'Other')
+     ])
+    marital = fields.Selection([
+         ('single', 'Single'),
+         ('married', 'Married'),
+         ('widower', 'Widower'),
+         ('divorced', 'Divorced')
+     ], string='Marital Status')
+    department_id = fields.Many2one('hr.department', string='Department')
+    address_id = fields.Many2one('res.partner', string='Working Address')
+    address_home_id = fields.Many2one('res.partner', string='Home Address')
+    bank_account_id = fields.Many2one('res.partner.bank', string='Bank Account Number',
+            domain="[('partner_id', '=', address_home_id)]", help='Employee bank salary account')
+    work_phone = fields.Char()
+    mobile_phone = fields.Char(string='Work Mobile')
+    work_email = fields.Char()
+    work_location = fields.Char()
+    notes = fields.Text()
+    parent_id = fields.Many2one('hr.employee', string='Manager')
+    category_ids = fields.Many2many('hr.employee.category',
+        'employee_category_rel', 'emp_id', 'category_id',
+        string='Tags')
+    child_ids = fields.One2many('hr.employee', 'parent_id', string='Subordinates')
+    resource_id = fields.Many2one('resource.resource', string='Resource',
+        ondelete='cascade', required=True, auto_join=True)
+    coach_id = fields.Many2one('hr.employee', string='Coach')
+    job_id = fields.Many2one('hr.job', string='Job Title')
+    # image: all image fields are base64 encoded and PIL-supported
+    image = fields.Binary(string='Photo',
+        help='This field holds the image used as photo for the employee, limited to 1024x1024px.')
+    image_medium = fields.Binary(string='Medium-sized photo', compute='_compute_image', store=True,
+        help='Medium-sized photo of the employee. It is automatically '\
+             'resized as a 128x128px image, with aspect ratio preserved. '\
+             'Use this field in form views or some kanban views.')
+    image_small = fields.Binary(string='Small-sized photo', compute='_compute_image', store=True,
+        help='Small-sized photo of the employee. It is automatically '\
+             'resized as a 64x64px image, with aspect ratio preserved. '\
+             'Use this field anywhere a small image is required.')
+    passport_id = fields.Char(string='Passport No')
+    color = fields.Integer(string='Color Index', default=0)
+    city = fields.Char(related='address_id.city')
+    login = fields.Char(related='user_id.login', readonly=True)
+    last_login = fields.Datetime(related='user_id.login_date',
+        string='Latest Connection', readonly=True)
+    active = fields.Boolean(default=True)
+
+    @api.depends('image')
+    def _compute_image(self):
+        for record in self:
+            record.image_medium = image.image_resize_image_medium(record.image)
```

Comment:
i think tools.image_resize_image_medium(record.image) no need to import image from tools.